client/lib/wp/sync-handler/index.js

```diff
@@ -186,6 +186,11 @@ export class SyncHandler {
 			 * @returns {Object} - combinedResponse object
 			 */
 			const cacheResponse = combinedResponse => {
+				debug( 'cacheResponse()', combinedResponse );
+				if ( ! combinedResponse ) {
```

Comment:
To be safe you probably also want to check if combinedResponse. serverResponse exists